### PR TITLE
Swift.Toolchain: add 5.5 release

### PR DESCRIPTION
**Pull request by compnerd**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28544)

### CODE REVIEW

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.5
```
Comment (ItzLevvie) [inline manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:3]:
```suggestion
PackageVersion: 0.0.0.0
```

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      MinimumVersion: 14.28.29913.0
+    - PackageIdentifier: Microsoft.WindowsSDK
+    - PackageIdentifier: Microsoft.VisualStudio.2019.BuildTools
+    - PackageIdentifier: Git.Git
+Installers:
+  - Architecture: x64
+    InstallerType: burn
+    InstallerUrl: https://swift.org/builds/swift-5.5-release/windows10/swift-5.5-RELEASE/swift-5.5-RELEASE-windows10.exe
+    InstallerSha256: 6da9d1c01b3f6c9bd7787d478b9ef670e64515875cc5afa035fd4bd751a7b6d2
+    ProductCode: '{74102b32-0048-48eb-b9ed-b477618d923f}'
```
Comment (ItzLevvie) [inline manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:30]:
```suggestion
    ProductCode: '{21af2387-06b6-4464-a951-ef49b8142b9e}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+PackageIdentifier: Swift.Toolchain
+PackageVersion: 5.5
+PackageName: Swift Toolchain
```
Comment (ItzLevvie) [inline manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:4]:
```suggestion
PackageName: Swift
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml (hunk 3 above)
Comment (ItzLevvie) [reply manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:4]: image

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml (hunk 2 above)
Comment (ItzLevvie) [reply manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:30]: image

File: manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:3]: image

Comment (compnerd) [reply manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:3]: This was discussed when we first imported the package - the package currently always indicates `0.0.0.0` for the version, though the toolchain version is different.

Comment (compnerd) [reply manifests/s/Swift/Toolchain/5.5/Swift.Toolchain.yaml:3]: https://github.com/microsoft/winget-pkgs/pull/19313#issuecomment-870860207

**Review by ItzLevvie — APPROVED** [review]

**Comment by svanimpe** [comment]
If I may also ask for some advice: Am I following the correct procedure here for patch version upgrades? That is: Should I create them as new versions with new manifests? Or should I edit the existing 5.5 manifest to point to 5.5.3? I'm not sure how the updater works, so I figured it may be best to create a new manifest for each version.

**Pull request merged**